### ISSUE TITLE
Why item border lines have different width when scale the view?

### ISSUE THREAD

**Issue by linkoln** [fralx/LimeReport#130]
![default](https://user-images.githubusercontent.com/5786476/50428515-45a49580-08f3-11e9-8c8e-c4bcd73421f7.png)

As you can see in the **red** and **blue** rect, it's strange that the border lines have different width when I scale the report view . I have test it in the latest code and the problem exists.Wish your help!

**Comment by fralx** [comment]
This happens because, in fact, 2 frames are superimposed on each other and the calculation of coordinates when scaling due to rounding does not take place accurately enough. The coordinates are calculated by the QGraphicsScene and i can't change it behavior :(. When you print or export a report, this will not happen. Maybe in the future i will create TableItem to prevent this behavior.

**Comment by linkoln** [comment]
Yeah, when print or export a report, it's no problem.
But i create a demo which implements my own QGraphicsItem and QGraphicsView and show the view,when i scale the view it don't have this problem. I'm confused with it.

**Comment by fralx** [comment]
Look at this pictures, it is a qgraphicscene example from qt. I just scale the picture and you can see the bold border exists on 1 screen and doesn't on second. 
![1](https://user-images.githubusercontent.com/14914730/50487589-0e77d680-0a10-11e9-9861-a6476099ff06.png)
![2](https://user-images.githubusercontent.com/14914730/50487591-10da3080-0a10-11e9-9d2a-0bd63c92bd96.png)

**Comment by linkoln** [comment]
You are right.So the only way to solve this problem is to change the QGraphicsScene 's source code?Have any way else?
